build.gradle

```diff
@@ -65,13 +65,13 @@ ext {
   ehcache2Version = '2.10.3'
 
   // Clustered
-  terracottaPlatformVersion = '5.3.0-pre35'
+  terracottaPlatformVersion = '5.3.1'
```

Comment:
Did we mean for platform and core versions to become misaligned?

Reply:
We never kept them aligned